bigchaindb/log/setup.py

```diff
@@ -0,0 +1,154 @@
+"""Setup logging."""
```

Comment:
Is there any chance that this file `setup.py` can be misinterpreted by some python packaging tools?